packages/base-controller/src/BaseControllerV2.ts

```diff
@@ -170,6 +175,26 @@ export class BaseController<
       nextState,
       patches,
     );
+
+    return { nextState, patches, inversePatches };
+  }
+
+  /**
+   * Rolls back controller state. Accepts an array of inverse patches
+   * that are applied to the current state. The inverse patches come from the
+   * update function itself.
+   *
+   * @param inversePatches - An array of immer patches that are to be applied to rollback
+   * changes made in a previous update.
+   */
+  protected rollback(inversePatches: Patch[]) {
```

Comment:
Should this be called `applyPatches` instead? It doesn't seem like there is any difference between a patch and an inverse patch.

Reply:
Yeah..that's true that there isn't a real difference. Hmm...maybe I should just make this a generic `applyPatches` function.

Reply:
I guess we can just call it "rollback" in the `SnapController` 😋